packages/server/integrations/TaskIntegrationManagerFactory.ts

```diff
@@ -42,11 +43,13 @@ export interface TaskIntegrationManager {
   )
 
   getIssue?(taskId: string)
+
+  getAPI?()
 }
 
 export default class TaskIntegrationManagerFactory {
   public static async initManager(
-    dataLoader: DataLoaderWorker,
+    dataLoader: DataLoaderWorker | RootDataLoader,
```

Comment:
+1 why add `RootDataLoader`? It looks like there's three uses of `TaskIntegrationManagerFactory` and they all use `DataLoaderWorker`